sdk/eventhub/event-hubs/samples/usingAadAuth.ts

```diff
@@ -9,13 +9,13 @@
     Please ensure that your Azure Event Hubs resource is in US East, US East 2, or West Europe
     region. AAD Role Based Access Control is not supported in other regions yet.
 
-    Register a new application in AAD and assign the "owner" role to it
+    Register a new application in AAD and assign the "Azure Event Hubs Data Owner (Preview)" role to it
```

Comment:
I haven't looked at the portal recently, maybe verify if the option still says "(Preview)" in the portal

Reply:
Yes, it says preview.